services/web-app/components/mdx-notification/mdx-notification.module.scss

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+@use '@carbon/react/scss/spacing' as spacing;
+@use '@carbon/react/scss/type' as type;
+@use '@carbon/react/scss/breakpoint' as breakpoint;
+@use '@carbon/react/scss/theme' as theme;
+@use '@carbon/react/scss/utilities/convert' as convert;
+
+// override carbon low contrast anchor
+:global(.cds--inline-notification:not(.cds--inline-notification--low-contrast) a) {
+  color: theme.$link-primary;
+}
+
+// override carbon tooltip
+:global(.cds--icon-tooltip) {
+  position: absolute;
+  top: 1px;
+  right: 1px;
+}
+
+// override carbon - icon inline with title
+:global(.cds--inline-notification__icon) {
+  margin-top: spacing.$spacing-05;
+}
+
+// override carbon max-width
+:global(.cds--inline-notification) {
+  max-width: 100%;
+}
+
+.error-container {
+  position: relative;
+  margin-top: spacing.$spacing-05;
+}
+
+.grid {
+  @include breakpoint.breakpoint-down(lg) {
+    position: relative;
+  }
+}
+
+.container {
+  display: inline;
+  width: 100%;
+}
+
+.title {
+  @include type.type-style('heading-compact-01', true);
+
+  padding-bottom: spacing.$spacing-01;
+  color: theme.$text-primary;
+}
+
+.link {
+  position: absolute;
+  top: convert.rem(8px);
+  right: convert.rem(8px);
+  padding: 0 spacing.$spacing-06;
+  color: theme.$link-primary;
+  cursor: pointer;
+}
+
+.description {
+  @include type.type-style('body-short-01', true);
+
+  padding-bottom: spacing.$spacing-05;
+  color: theme.$text-primary;
+}
+
+.content {
+  @include type.type-style('code-02', true);
+
+  color: theme.$text-primary;
+
+  overflow-x: auto;
+  white-space: nowrap;
+
+  @include breakpoint.breakpoint-down(lg) {
+    margin-bottom: spacing.$spacing-07;
+  }
+}
+
+.collapse-icon {
+  z-index: 1;
+
+  &:hover {
+    background-color: rgba(0, 157, 154, 0.1);
```

Comment:
just double checking there isn't a token for this?

Reply:
I can ask - I didn't see one :/

Reply:
no worries, just wanted to double check

Reply:
I think anna said let's go with `$background-hover`, especially since the notification isn't necessarily doing to be info kind (blue)